deploy/README.md

```diff
@@ -68,6 +69,26 @@ The diagram below illustrates the components of the Kubernetes collection soluti
 * **Fluentd deployment.** Forwards logs and metrics to HTTP sources on a hosted collector. Includes multiple Fluentd plugins that parse and format the metrics and enrich them with metadata.
 * **Events fluentd deployment.** Forwards events to an HTTP source on a hosted collector.
 
+## Support Matrix
+
+The following matrix displays the minimum supported version of required services.
```

Comment:
@drduke1 let's sync offline for this. I talked to @frankreno and we have some idea about the structure.

Reply:
Okay, I was following guidance as instructed already from https://docs.google.com/document/d/1TU-QyM7yFl4KFLrGjZ8nCL6Li2NCn0I3dmwx7H8wgrA/edit?usp=sharing

Reply:
Talked to @samjsong . He will update the table there